certbot/certbot/_internal/main.py

```diff
@@ -495,29 +484,65 @@ def _report_new_cert(config, cert_path, fullchain_path, key_path=None):
 
     """
     if config.dry_run:
-        _report_successful_dry_run(config)
+        display_util.notify("The dry run was successful.")
         return
 
     assert cert_path and fullchain_path, "No certificates saved to report."
 
     expiry = crypto_util.notAfter(cert_path).date()
-    reporter_util = zope.component.getUtility(interfaces.IReporter)
-    # Print the path to fullchain.pem because that's what modern webservers
-    # (Nginx and Apache2.4) will want.
-
-    verbswitch = ' with the "certonly" option' if config.verb == "run" else ""
-    privkey_statement = 'Your key file has been saved at:{br}{0}{br}'.format(
-            key_path, br=os.linesep) if key_path else ""
-    # XXX Perhaps one day we could detect the presence of known old webservers
-    # and say something more informative here.
-    msg = ('Congratulations! Your certificate and chain have been saved at:{br}'
-           '{0}{br}{1}'
-           'Your cert will expire on {2}. To obtain a new or tweaked version of this '
-           'certificate in the future, simply run {3} again{4}. '
-           'To non-interactively renew *all* of your certificates, run "{3} renew"'
-           .format(fullchain_path, privkey_statement, expiry, cli.cli_command, verbswitch,
-               br=os.linesep))
-    reporter_util.add_message(msg, reporter_util.MEDIUM_PRIORITY)
+
+    display_util.notify(
+        ("\nSuccessfully received certificate.\n"
+        "Certificate is saved at: {cert_path}\n{key_msg}{symlink_msg}"
+        "This certificate expires on {expiry}.").format(
+            cert_path=fullchain_path,
+            expiry=expiry,
+            key_msg="Key is saved at:         {}\n".format(key_path) if key_path else "",
+            symlink_msg="These files will be automatically updated "
```

Comment:
I assume this is separate so we can eventually print something conditionally based on the system automatic renewal flag?

Reply:
I see no reason not to vary this by `preconfigured_renewal` in this PR. I'll think of some wording.

Reply:
How verbose do we want to get here? How verbose is verbose enough to solve #8327?

There is probably a big difference between "Certbot will automatically renew this certificate" vs "Certbot's preconfigured  renewal task will automatically renew this certificate".

So far I have:

With `--preconfigured-renewal`:

    Successfully received certificate.
    Certificate is saved at: /etc/letsencrypt/live/example.com/fullchain.pem
    Key is saved at:         /etc/letsencrypt/live/example.com/privkey.pem
    This certificate expires on 2025-12-23.
    Certbot will automatically renew this certificate and keep these files up-to-date.

Without:

    Successfully received certificate.
    Certificate is saved at: /etc/letsencrypt/live/example.com/fullchain.pem
    Key is saved at:         /etc/letsencrypt/live/example.com/privkey.pem
    This certificate expires on 2025-12-23.
    These files will be updated when the certificate renews. Run "certbot renew" to renew all expiring certificates.

Unfortunately neither is very explicit about whether the user needs to do something (create a scheduled task) or not. 

If the user followed the instructions generator, it's probably fine.

Reply:
For with `--preconfigured-renewal`, maybe explicitly say something along the lines of "you do not need to take any further action" just to be extra clear?

For without, you're right about the instruction generator case, we're set there. Debating if we want to add a line like "we recommend creating a scheduled task to run certbot renew every 60 days". We could also copy the instructions from the generator directly inline, or just link to them. Or just leave it? Ideally the number of people who would ever see that message will decrease over time.

Thinking of that article we read, I feel like they'd say "you should tell them what the next thing to run is." What was your reasoning in landing on these messages? I don't have a strong sense of what the right thing to do here is.

> Another problem is that the messaging is untrue if the user has used --manual without authentication hooks. There are still a lot of users who rock up asking why certbot renew complains about their manually-acquired certificates. Do we want to solve that first? It's a can of worms but I think is probably a major UX problem that's worth addressing.

I agree this is worth addressing. I suspect it's enough of its own issue to be worth addressing separately, in a follow-up PR.

Reply:
>I agree this is worth addressing. I suspect it's enough of its own issue to be worth addressing separately, in a follow-up PR.

Just linking #6280 so I don't forget this exists.

Reply:
>something along the lines of "you do not need to take any further action" just to be extra clear?

I was down with this, but when adding it I realized it might be a bit ambiguous for some users. Like, "you do not need to install your certificate" if the verb was `certonly`.

I've gone with the below, wdyt. I like that it's succinct and I don't think it's lost any meaning:

>Certbot will automatically renew this certificate in the background.

Reply:
I love that.